ignite/distributed/comp_models/xla.py

```diff
@@ -140,7 +140,7 @@ def _do_all_reduce(self, tensor: torch.Tensor, op: str = "SUM", group: Optional[
             if group is not None and not self._check_group_type(group):
                 raise ValueError("Argument group should be list of int")
             op = self._reduce_op_map[op]
-            xm.all_reduce(op, [tensor], groups=group)
+            xm.all_reduce(op, [tensor], groups=[group])
```

Comment:
This is unrelated to this PR, please send that in another PR

Reply:
https://github.com/pytorch/ignite/pull/2948